youtube_dl/extractor/neteasemusic.py

```diff
@@ -84,7 +84,8 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'id': '32102397',
             'ext': 'mp3',
             'title': 'Bad Blood (feat. Kendrick Lamar)',
-            'creator': 'Taylor Swift / Kendrick Lamar',
```

Comment:
Redundant or not it should be preserved either by providing manually or by copying from `artist` automatically when not present (that is not implemented) since it can be used by someone already.